sydent/config/general.py

```diff
@@ -74,14 +71,16 @@ def parse_config(self, cfg: "ConfigParser") -> None:
 
         self.address_lookup_limit = cfg.getint("general", "address_lookup_limit")
 
-        self.prometheus_enabled = cfg.has_option("general", "prometheus_port")
-        if self.prometheus_enabled:
-            self.prometheus_port = cfg.getint("general", "prometheus_port")
-            self.prometheus_addr = cfg.get("general", "prometheus_addr")
+        self.prometheus_port = cfg.getint("general", "prometheus_port", fallback=None)
```

Comment:
to confirm: have you checked that `getint` is happy with `fallback=None` as opposed to an integer?

Reply:
Yep!

Reply:
Yep! At least for now: https://github.com/python/cpython/blob/17a1b3e63a5db4e581c0ef52751df2314f7249fa/Lib/configparser.py#L806